tests/Command/ExpirePasswordTest.php

```diff
@@ -68,12 +68,14 @@ public function testExpirePasswordUserNotExisting() {
 			->with('not-existing-uid')
 			->willReturn(null);
 
-		$this->commandTester->execute([
-			'uid' => 'not-existing-uid',
-			'expiredate' => '2018-06-28 10:13 UTC'
-			]);
+        $returnCode = $this->commandTester->execute([
```

Comment:
tabs

Reply:
Saw that. Took a little bit to change it.